mongo/dataset_store.go

```diff
@@ -194,7 +204,12 @@ func (m *Mongo) GetVersions(id, editionID, state string) (*models.VersionResults
 	selector := buildVersionsQuery(id, editionID, state)
 
 	iter := s.DB(m.Database).C("instances").Find(selector).Iter()
-	defer iter.Close()
+	defer func() {
+		err := iter.Close()
+		if err != nil {
+			log.ErrorC("error closing iterator", err, log.Data{"selector": selector})
```

Comment:
this is identical to the error on line 103 - is it worth changing the text to help narrow-down where this error comes from? `"error closing instances iterator"` or is it clear from `selector`?

Reply:
good spot 👍, i updated the log line and the other log line as well